pkgs/tools/security/ghidra/doc/tests/gui-01.nix

```diff
@@ -0,0 +1,16 @@
+# Expected result:
+# A Ghidra GUI instance should start with the scala loader plugin listed under 
+#  File -> Install Extensions
+let pkgs = import <nixpkgs> {}; in
```

Comment:
this would import nixpkgs from the user's NIX_PATH rather then the current repository.

Reply:
What's the correct way to do it?

Reply:
You can use a relative path like:

```
let pkgs = import ../../../../../../ {}; in
```

Reply:
A few days later, it hit me what you wanted. It feels kind of dirty but it makes sense.